apm-commons/apm-datacarrier/src/test/java/org/apache/skywalking/apm/commons/datacarrier/consumer/ConsumerTest.java

```diff
@@ -35,6 +35,7 @@
 
     @Test
     public void testConsumerLessThanChannel() throws IllegalAccessException {
+        BUFFER.drainTo(new ArrayList<SampleData>());
```

Comment:
Seems `BUFFER` is actually not used in `testConsumerLessThanChannel`? Can `BUFFER` be simply moved into `testConsumerMoreThanChannel` as a local variable?

Reply:
good catch! removed